OverPassLayer.js

```diff
@@ -229,6 +244,11 @@ L.OverPassLayer = L.FeatureGroup.extend({
           var queryWithMapCoordinates = this.options.query.replace(/(BBOX)/g, bbox.toOverpassBBoxString());
           var url =  this.options.endpoint + "interpreter?data=[out:json];" + queryWithMapCoordinates;
 
+          if (beforeRequest) {
```

Comment:
ah - `beforeRequest == before_a_set_of_requests` ^^
but than it's possible that `beforeRequest()` is not triggered. Imagine the first bbox was already requested. What is the reason of placing  the `beforeRequest()` call here and not at the start of `onMoveEnd`?